.github/workflows/CD.yaml

```diff
@@ -60,6 +60,8 @@ jobs:
           TRUSTEE_CREDENTIALS: ${{ secrets.TRUSTEE_CREDENTIALS }}
       - name: check deployment
         run: ./gradlew :dataland-frontend:npm_run_testpreview --no-daemon --stacktrace
+        env:
+          ELECTRON_EXTRA_LAUNCH_ARGS: "--ignore-connections-limit=localhost:8090,proxy,localhost,preview-dataland.duckdns.org,dev-dataland.duckdns.org"
```

Comment:
können wir feststellen, dass diese Änderungen wirklich einen effekt hatten in der CI?